api/src/main/java/io/strimzi/api/kafka/model/AbstractKafkaConnectSpec.java

```diff
@@ -43,6 +44,8 @@ public abstract class AbstractKafkaConnectSpec extends Spec implements HasConfig
     private Tracing tracing;
     private KafkaConnectTemplate template;
     private ExternalConfiguration externalConfiguration;
+    private String clientRackInitImage;
```

Comment:
I was wondering if we want to be so explicit about the name of this init image used just for client rack.
I mean, now it's just client rack but who knows the future and if that could be used for some other pre-processing.
Can't we think about a more generic name? Just `initImage` or something like that?

Reply:
You cannot change this. It is part of the Connect API already.

Reply:
That's true, and it is the reason I didn't touch it. We could create a better name deprecating the old one, but I would do it only when we will actually need to do multiple things during the init phase.

Reply:
Damn! How was it used and why named "clientRack" if we are adding the rack-awareness now?

Reply:
MM2 rack awareness feature is built on top of KafkaConnect's one, that's why.

Reply:
Connect supports it for a long time. This PR only adds the support to MM2. In theory, you can have the configuration completely separate and use different options for each. But given that we do not have any plans for the init container to do anything else (I mean not just no plans to add something this month, but I'm not aware of any feature which would do it) I would stick with what Federico did.